src/screens/SendCoinsScreen.tsx

```diff
@@ -618,7 +634,14 @@ export class SendCoinsScreen extends Component<Props, State> {
             <TouchableOpacity style={styles.addressBookIcon}>
               <Image style={styles.icon} source={images.addressBook} />
             </TouchableOpacity>
-            <TouchableOpacity style={styles.qrCodeIcon}>
+            <TouchableOpacity
+              style={styles.qrCodeIcon}
+              onPress={() =>
+                this.props.navigation.navigate(Route.ScanQrAddress, {
+                  onAddressRead: this.processAddressData,
+                })
```

Comment:
So the usage would be:
```
    this.props.navigation.navigate(Route.ScanQrCode, {
      onBarCodeScan: this.processAddressData,
    });
```

But the `ScanQrCodeScreen` is missing that code:
```
 if (RNCamera.Constants.CameraStatus === RNCamera.Constants.CameraStatus.READY) this.cameraRef.pausePreview();
    if (+new Date() - this.lastTimeIveBeenHere < SCAN_CODE_AFTER_MS) {
      this.lastTimeIveBeenHere = +new Date();
      return;
    }
    this.lastTimeIveBeenHere = +new Date();
```

but if it's needed it could be added.

Reply:
Oh, I haven't noticed. Completely right. Thanks